### DISCUSSION TITLE
A number of PRs are stuck

### DISCUSSION THREAD

**Discussion by cesarcoatl** [Homebrew/discussions#385] [Casks] [answered]
The oldest PR has been stuck for more than 12 hours, which is somewhat atypical.

97033, 97034, 97035, and 97036.

Two checks are pending on all: syntax and test.

**Comment by SMillerDev** [comment, marked as answer]
The best way is just to make a comment in the PR, asking someone to restart CI.

**Reply by k4rtik** [reply]
Is there a way to ask the bot to restart it? This one is stuck: https://github.com/Homebrew/homebrew-cask/pull/97279

Failed with:
> This request was automatically failed because there were no enabled runners online to process the request for more than 1 days.

**Reply by cesarcoatl** [reply]
I've tried step 7 from this guide: [Requesting a pull request review](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review).

Not sure if it was pure magic, but it worked on one of my PRs that failed at the syntax step. See: [Homebrew/hombrew-cask-versions/#10164](https://github.com/Homebrew/homebrew-cask-versions/pull/10164#event-4168339284)

Additionally, see discussion #442.

**Reply by k4rtik** [reply]
I tried step 7, don't think that's working. It seems like a different issue where the CI has timed out as no workers were available.

This used to be a much faster process when I last sent a PR: https://github.com/Homebrew/homebrew-cask/pull/88306 took less than an hour for merge. :(

**Reply by jonchang** [reply]
Folks, this is a Github issue. There is not a whole lot we can do until their support gets back to us.
